app/src/main/java/org/akvo/flow/service/ExceptionReportingService.java

```diff
@@ -153,6 +153,8 @@ public void submitStackTraces(String server) {
                     File f = new File(dir, list[i]);
                     String trace = FileUtil.readFileAsString(f);
 
+                    // We cannot use the standard FlowApi.getDeviceParams, fot this service uses
+                    // a different naming convention...
                     Map<String, String> params = new HashMap<String, String>();
```

Comment:
So we don't introduce the `ANDROID_ID` here because of the convention or because its not needed?